src/Microsoft.DncEng.SecretManager/Readme.md

```diff
@@ -100,6 +100,84 @@ This step runs the `synchronize --verify-only` command before the "approval" sta
 ### Weekly Rotation
 This step runs the `synchronize` command. This compares every secret specified in the manifest with the corresponding vault, and performs the appropriate rotation for each secret that requires it. Failures in this step are caused by secrets that require rotation and can't be rotated automatically. To manually rotate these secrets a human will need to run the `synchronize` command manually and specify the manifest that contains the secret requiring rotation.
 
+## Onboarding a new Repo
+- if .config/dotnet-tools.json doesn't exist
+  - dotnet new tool manifest
+- dotnet tool install microsoft.dnceng.secretmanager --version 1.1.0-*
+- create manifests for vaults in .vault-config
```

Comment:
Should specify where .vault-config should go (the root of the repository? the eng folder? etc.). Since the path can be arbitrary, I'd note that it has to match the path in the scripts below.